keda/templates/NOTES.txt

```diff
@@ -59,6 +59,12 @@ WARNING - Running on unsupported Kubernetes version "1.{{.Capabilities.KubeVersi
 -------------------------------------------------------------------------------------
 {{- end }}
 
+{{- if .Values.serviceAccount.name }}
+-------------------------------------------------------------------------------------
+WARNING - .serviceAccount.name has been deprecated, please migrate to newest version of the Helm Chart values that allows overriding the service account name for each KEDA component
```

Comment:
Let's list the knobs they can use in the new version

Reply:
done in https://github.com/kedacore/charts/pull/625/commits/1ad805c673c72c6325196df1408be7acd8db8b14